pkg/clients/ocm/client.go

```diff
@@ -10,20 +10,34 @@ import (
 type Environment string
 
 const (
-	Production  Environment = "https://api.openshift.com"
-	Stage       Environment = "https://api.stage.openshift.com"
-	Integration Environment = "https://api.integration.openshift.com"
+	Production         Environment = "https://api.openshift.com"
+	Stage              Environment = "https://api.stage.openshift.com"
+	Integration        Environment = "https://api.integration.openshift.com"
+	FedRampProduction  Environment = "https://api.openshiftusgov.com"
+	FedRampStage       Environment = "https://api.stage.openshiftusgov.com"
+	FedRampIntegration Environment = "https://api.int.openshiftusgov.com"
 )
 
 type Client struct {
 	*ocmsdk.Connection
 }
 
-func New(ctx context.Context, token string, environment Environment) (*Client, error) {
-	connection, err := ocmsdk.NewConnectionBuilder().
-		URL(string(environment)).
-		Tokens(token).
-		BuildContext(ctx)
+func New(ctx context.Context, token string, clientID string, clientSecret string, environment Environment) (*Client, error) {
+	var connection *ocmsdk.Connection
+	var err error
+
+	if clientID != "" && clientSecret != "" {
+		connection, err = ocmsdk.NewConnectionBuilder().
+			URL(string(environment)).
+			Client(clientID, clientSecret).
+			BuildContext(ctx)
+	} else {
+		connection, err = ocmsdk.NewConnectionBuilder().
+			URL(string(environment)).
+			Tokens(token).
+			BuildContext(ctx)
+	}
```

Comment:
```suggestion
    connectionBuilder, err = ocmsdk.NewConnectionBuilder().URL(string(environment))
    if clientID != "" && clientSecret != "" {
        connectionBuilder = connectionBuilder.Client(clientID, clientSecret)
    } else {
        connectionBuilder = connectionBuilder.Tokens(token)
    }
    connection, err := connectionBuilder.BuildContext(ctx)
```
something like this is slightly cleaner, using the builder object like in java!